gradle.properties

```diff
@@ -0,0 +1,2 @@
+http.socketTimeout=20000
+http.connectionTimeout=20000
```

Comment:
I think the property names have been changed? see https://github.com/gradle/gradle/blob/master/subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/JavaSystemPropertiesHttpTimeoutSettings.java#L26